lib/galaxy/tools/__init__.py

```diff
@@ -2791,13 +2814,18 @@ def produce_outputs(self, trans, out_data, output_collections, incoming, history
 
             copied_value = dce_object.copy()
```

Comment:
This line is not needed any more, I think.

Reply:
Indeed!

Reply:
Still present.

Reply:
There was another one in this rebase at 2526 - must have been a repeated copy and paste mistake and just removed one instance. I'll rebase again and try to get them all.